types/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "module": "esnext",
```

Comment:
Would it be useful to keep the `module` and `lib` options as:
```json
{
  "module": "commonjs",
  "lib": ["es2015"]
}
```

Reply:
Ok. But I don't think it matters - tsconfig is only needed for linting types

Reply:
Gotcha, I'm ok with leaving it as is then